### PR TITLE
Update encryption.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Teams to the list

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1650

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/encryption.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ With Office 365, multiple layers and kinds of encryption work together to secure
...
 |Email in transit between recipients. This email includes email hosted by Exchange Online.  <br/> |Office 365 Message Encryption with Azure Rights Management, S/MIME, and TLS for email in transit  <br/> |[Office 365 Message Encryption (OME)](ome.md) <br/> [Email encryption in Office 365](email-encryption.md) <br/> [How Exchange Online uses TLS to secure email connections in Office 365](exchange-online-uses-tls-to-secure-email-connections.md) <br/> |
+|Chats, Messages and Files in transit between recipients using Microsoft teams <br/> |Teams uses TLS and MTLS to encrypt instant messages. Media traffic is encrypted using Secure RTP (SRTP). Teams uses FIPS (Federal Information Processing Standard) compliant algorithms for encryption key exchanges <br/> |[Encryption for MS Teams](https://docs.microsoft.com/en-us/microsoftteams/teams-security-guide#encryption-for-teams) <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/encryption.md]:
```suggestion
|Chats, messages, and files in transit between recipients using Microsoft Teams. <br/> |Teams uses TLS and MTLS to encrypt instant messages. Media traffic is encrypted using Secure RTP (SRTP). Teams uses FIPS (Federal Information Processing Standard) compliant algorithms for encryption key exchanges. <br/> |[Encryption for Teams](https://docs.microsoft.com/microsoftteams/teams-security-guide#encryption-for-teams) <br/> |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@KCCross - PR has been copy edited and is ready for final review. This is based on issue #1650.

Thank you.

cc: @kenwith

**Comment by mypil** [comment]
@KCCross - We would like to follow up. Can you please assist to review and merge this PR?

Thank you.

**Pull request merged**